### PR TITLE
feat(ui): Render errored threads in issue details

### PR DESCRIPTION
**Pull request by jan-auer**
We're introducing a new thread attribute called `errored`. It indicates that a thread is the cause of an error, without having crashed the application. Formally, the UI would only handle `crashed` threads. 

This extends the UI to also consider threads with `errored`, if there is no crashed thread. Such a thread is marked specially in the thread list, and selected by default.

See https://github.com/getsentry/semaphore/pull/306

### CODE REVIEW

**Comment by jan-auer** [comment]
This still requires an acceptance test, which @marandaneto is helping me to create right now.

**Comment by jan-auer** [comment]
Store normalization at the moment still deletes the `errored` attribute in the integration test. Once `libsemaphore` is released, we can bump and should see correct snapshots.

File: src/sentry/static/sentry/app/components/events/interfaces/threads.jsx [hunk 1]
```diff
@@ -139,14 +135,20 @@ class Thread extends React.Component {
...
+
     return (
       <div className="traceback missing-traceback">
         <ul>
           <li className="frame missing-frame">
             <div className="title">
-              <span className="informal">
-                {this.props.data.crashed ? 'Thread Crashed' : 'No or unknown stacktrace'}
-              </span>
+              <span className="informal">{t(message)}</span>
```
Comment (bruno-garcia) [inline src/sentry/static/sentry/app/components/events/interfaces/threads.jsx]: I've seen the `No or unknown stacktrace` a few times, I found useful to point out the thread has no frames.

**Review by bruno-garcia — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/interfaces/threads.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/events/interfaces/threads.jsx]: `t()` only works on static strings

File: src/sentry/static/sentry/app/components/events/interfaces/threads.jsx [hunk 2]
```diff
@@ -139,14 +135,20 @@ class Thread extends React.Component {
   };
 
   renderMissingStacktrace = () => {
+    const {crashed, errored} = this.props.data;
+    let message = 'No or unknown stacktrace';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/threads.jsx]: These should be wrapped with `t()`

File: src/sentry/static/sentry/app/components/events/interfaces/threads.jsx [hunk 3]
```diff
@@ -113,17 +114,12 @@ function getIntendedStackView(thread, event) {
...
-    }
-  }
-  for (const thread of threads) {
-    if (thread.stacktrace) {
-      return thread;
-    }
-  }
-  return threads[0];
+  return (
+    threads.find(thread => thread.crashed) ||
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/threads.jsx]: We maybe want a comment here that there is a priority involved with finding the "best" thread (I could see someone in the future trying to simplify this as finding any crashed|errored|stacktrace thread.

Comment (jan-auer) [reply src/sentry/static/sentry/app/components/events/interfaces/threads.jsx]: Good idea, done.

**Pull request merged**